include/openssl/crypto.h

```diff
@@ -86,33 +86,48 @@ OPENSSL_EXPORT int CRYPTO_needs_hwcap2_workaround(void);
 
 // Data-Independent Timing (DIT) on AArch64
 
-#if defined(OPENSSL_AARCH64) && !defined(OPENSSL_WINDOWS) && defined(MAKE_DIT_AVAILABLE)
+#if defined(OPENSSL_AARCH64) && !defined(OPENSSL_WINDOWS)
 // (TODO): See if we can detect the DIT capability in Windows environment
 
-// armv8_enable_dit sets the DIT flag to 1 and returns its original value
+// armv8_set_dit sets the DIT flag to 1 and returns its original value
 // before it was called.
-uint64_t armv8_enable_dit(void);
+OPENSSL_EXPORT uint64_t armv8_set_dit(void);
```

Comment:
do we need to make the caller track this? can we store it ourselves as a thread-local variable?

Reply:
IMO, `armv8_set_dit` and `armv8_restore_dit` should be internal-only implementation details. They shouldn't be declared in this file.